### ISSUE TITLE
More guidance / notice on roadmap and breaking changes

### ISSUE THREAD

**Issue by isaacabraham** [NuGet/NuGetGallery#4088]
At the moment, the roadmap and plans for Nuget server is difficult to know. Keeping up with changes to the server API if you're working on a client that interfaces with it is hard e.g. Paket, Rider, MyGet, AppVeyor etc. etc.

As an example, there's a change coming up to the server API for SemVer2 support which we on the Paket team only found out about purely by chance. It would really help if the same sort of information flow that goes to the NuGet *client* team would go out publicly so all products that depend on the server API could know in advance what's upcoming, rather than just having to find out further down the road etc..

Even better would be to allow others to join in the discussion for features when they're just at the design stage - perhaps we'll have some great ideas that can make them even better. As an tiny example, Paket already found an issue in the SemVer2 support - which has now been fixed, so a win for everyone.

Cheers

**Comment by anangaur** [comment]
@isaacabraham We did have a [spec ](https://github.com/NuGet/Home/wiki/SemVer2-support-for-nuget.org-%28server-side%29) out for discussion quite some time before we started to execute on it. But I guess we can do better than that. I am discussing with the team on how to make these specs and future work more visible.

**Comment by isaacabraham** [comment]
Great, that would be a good start. I notice (unless I've misunderstood) that there was a breaking change pushed out today / yesterday that broke Paket as well :( Thanks to Nuget team to helping to help resolve the issue but again forward notice / discussions would really help :)

**Comment by dtretyakov** [comment]
@anangaur, your team could use the same approach as dotnet team - create a [repo for announcement](https://github.com/dotnet/announcements), where in issues you could share proposals which could be discussed by community.

**Comment by maartenba** [comment]
@dtretyakov An announcements repo would be a fantastic addition, although it would also have to go beyond a reactive announcements repo ("after the fact" announcements). Ideally it would also announce specs that are still in progress. This allows better community participation in the design/implementation process if anyone out there wants to get involved.

**Comment by anangaur** [comment]
@isaacabraham @dtretyakov @maartenba 
Here is the https://github.com/NuGet/Announcements repo. The intention is not just "after the fact" announcements but also to notify new & important features before the implementation. Especially the ones with [breaking changes](https://github.com/nuget/Announcements/labels/Breaking%20Change).

We will soon be announcing the new features once they are ready for [reviewing](https://github.com/nuget/Announcements/labels/Reviewing)

**Comment by maartenba** [comment]
Thumbs up! Can existing ones announced in the latest blog post be added there as well?

**Comment by anangaur** [comment]
Yes ! As they become ready for review.

**Comment by anangaur** [comment]
@isaacabraham I am closing this issue since we have [NuGet/announcements](https://github.com/nuget/announcements) repo for this purpose.
